lib/include/jxl/codestream_header.h

```diff
@@ -346,32 +348,47 @@ typedef struct {
 } JxlBlendInfo;
 
 /** The information about layers.
- * If coalescing is enabled (default), this can be ignored.
+ * When decoding, if coalescing is enabled (default), this can be ignored.
+ * When encoding, these settings apply to the pixel data given to the encoder,
+ * the encoder could choose an internal representation that differs.
  */
 typedef struct {
-  /** Horizontal offset of the frame (can be negative, always zero if coalescing
-   * is enabled)
+  /** Whether cropping is applied for this frame. If false, crop_x0, crop_y0
+   * are assumed to be zero, and xsize and ysize are assumed to match the main
```

Comment:
I would say for encoding crop_x0/y0 and xsize/ysize are ignored if have_crop is false, and for decoding they will be equal to the image dimensions in that case.

"are assumed to be" is a bit vague...

Reply:
Done